### ISSUE TITLE
haskell-ng: utf8-string wrong sha

### ISSUE THREAD

**Issue by manpages** [NixOS/nixpkgs#5920]
ditto

**Comment by peti** [comment]
The Cabal file was edited on Hackage. I'll commit an update in a little while.

**Comment by peti** [comment]
BTW, when an `editedCabalFile` hash is out of sync, then you can determine the correct one by running:

    curl http://hackage.haskell.org/package/utf8-string-0.3.8/utf8-string.cabal | sha256sum

It's okay to edit `hackage-packages.nix` manually for this kind of update if it's urgent.

**Comment by manpages** [comment]
@peti, I know, I did that, I just didn't know if I should post this here. From what I understand I shouldn't even send a PR with correct hash because you have a script for it?

**Comment by peti** [comment]
@manpages, yes, exactly. The `hackage2nix` utility generates all those checksums automatically; so there's no need to go through a PR if you find an outdated one.

**Comment by manpages** [comment]
Oh, forgot about hackage2nix! 

I'll refer to this thread if anyone I know has questions regarding hash mismatches in haskellng.

By the way, I've built *a lot* of packages on my system (sadly didn't keep track of the ones that built successfully, silly me) and I want to say that haskellng looks pretty solid at the moment, I've got several haskellEnvs with all that I need to finally be a happy haskell programmer who escaped the cabal hell. Hopefully, I'll find ways to contribute back to make haskellng "cabal heaven".

:heart:
